scripts/start_civ.sh

```diff
@@ -55,6 +55,8 @@ GUEST_STATIC_OPTION="\
  -device qemu-xhci,id=xhci \
  -usb \
  -device usb-kbd \
+ -device usb-host,vendorid=0x046d,productid=0x85c,id=id0 \
+ -device usb-host,vendorid=0x046d,productid=0x85c,id=id1 \
```

Comment:
Do not use hardcode in this script.
Please use it like: start_civ.sh -e "-device usb-host,vendorid=0x046d,productid=0x85c,id=id0 -device usb-host,vendorid=0x046d,productid=0x85c,id=id1"

If you want to support general usb camera passthrough, please add an option `"--passthrough-usb-camera"` which should enumerate usb camera from the platform and passthrough them to guest.